### PR TITLE
[LEVWEB-1202] Add more detailed CP records to mock

### PR DESCRIPTION
**Pull request by RobinKnipe**
...from the changes to the `lev-wrangler`.

### CODE REVIEW

File: mock/partnership_registration_v1.json [hunk 1]
```diff
@@ -71,31 +71,31 @@
         "surname": "FATHER",
         "occupation": "Father",
         "retired": true,
-        "designation": null,
+        "designation": "Undesignated",
```
Comment (daniel-ac-martin) [inline mock/partnership_registration_v1.json]: I think desgnation's might be specific things. Though I can't think what. Might be worth checking the test data.

**Comment by daniel-ac-martin** [comment]
Would be nice to keep this stuff in-line with the marriages data but I suppose we can worry about that later.

File: mock/partnership_registration_v1.json (hunk 1 above)
Comment (RobinKnipe) [reply mock/partnership_registration_v1.json]: They could well be set things, but the fields are just text, so all I'm really worried about is the value being included.

**Comment by RobinKnipe** [comment]
Yes agreed - I've added the `example-db` changes.

File: mock/partnership_registration_v1.json (hunk 1 above)
Comment (daniel-ac-martin) [reply mock/partnership_registration_v1.json]: The problem is that it makes the test data less realistic overall.

Comment (RobinKnipe) [reply mock/partnership_registration_v1.json]: Fixed

**Pull request merged**